crates/bitwarden/src/auth/password/strength.rs

```diff
@@ -13,7 +14,7 @@ pub(crate) fn password_strength(
     let mut arr: Vec<_> = inputs.iter().map(String::as_str).collect();
     arr.extend(GLOBAL_INPUTS);
 
-    zxcvbn(&password, &arr).map_or(0, |e| e.score())
+    zxcvbn(password.expose(), &arr).map_or(0, |e| e.score())
```

Comment:
I guess `zxcvbn` will create copies on their end?

Reply:
They make at least one yeah, this is inside the `zxcvbn` function:
```
let password = password.chars().take(100).collect::<String>();
```

I'll look later into seeing how that can be improved.